core/trino-main/src/main/java/io/trino/execution/StageStats.java

```diff
@@ -187,7 +188,8 @@ public StageStats(
 
```

Comment:
add `TestStageStats` tests

Reply:
First, I want to have an agreement with the direction of this PR.